tools/unit-tests/gen-test-fdlibm.c

```diff
@@ -0,0 +1,755 @@
+/* Copyright 2016 Samsung Electronics Co., Ltd.
+ * Copyright 2016 University of Szeged.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Unit test generator for fdlibm.
+ * To be compiled separately from the rest of jerry and to be linked to a trusted libm.
+ * Its output should be redirected to test-fdlibm.inc.h.
+ *
+ * Example:
+ * gcc gen-test-fdlibm.c -o gen-test-fdlibm -lm
+ * ./gen-test-fdlibm >test-fdlibm.inc.h
+ */
+
+#include <math.h>
+#include <stdio.h>
+
+#define GEN_INT_TEST(EXPR) printf("check_int (\"%s\", %s, %d);\n", #EXPR, #EXPR, EXPR);
+#define GEN_DBL_TEST(EXPR) printf("check_double (\"%s\", %s, %.20E);\n", #EXPR, #EXPR, EXPR);
+
+int
+main (int argc, char **args)
+{
+  printf ("/*\n"
+          " * Generated by tools/gen-test-fdlibm.sh\n"
+          " * DO NOT EDIT!!!\n"
+          " */\n");
+
+  /* acos tests */
+  GEN_DBL_TEST (acos (0.0));
```

Comment:
Do I understand correctly that these values are just hand picked values? Or do they come from an available corner case testing environment?